source/overview/license-and-subscription.rst

```diff
@@ -0,0 +1,138 @@
+Licensing and Subscription
+----------------------
+
+Mattermost offers a `secure self-service Customer Portal <https://customers.mattermost.com>`_ where you can easily purchase and manage your Mattermost Enterprise subscriptions.
+
+When you purchase a subscription, a license is generated. This license gives you access to Mattermost's Enterprise features. Subscriptions purchased via the Customer Portal are stored and listed in the portal, with license details including their start and end dates, providing you with an at-a-glance overview of your account. You also have full access to your billing history  making it easier to manage purchases and renewal dates.
+
+You can access your Customer Portal account to view information about your:
+
+- Subscription purchases
+- Licenses
+- Customer Portal account password
+- Organization information
+- Payment methods
+- Renewals (available in a future release)
+- Active users (available in a future release)
+
+Purchasing an Enterprise license
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**If you don't have a Customer Portal account**
+
+Open the `Customer Portal page <https://customers.mattermost.com>`__. Enter the required information and check the box to confirm **I have read and agree to the Privacy Policy**. Then select **Next**. When you've completed the process and validated your email address, you can proceed with the steps below. 
+
+**If you have an existing Customer Portal account**
+
+1. Log in to your `Customer Portal <https://customers.mattermost.com>`_ account. 
+2. Choose a subscription type and enter the number of users in the **Order summary** field.
+   - This indicates the number of users you can have on this subscription's instance. For more information about how users are defined, see our `FAQ <https://about.mattermost.com/pricing/#faq>`_. 
```

Comment:
This is not rendering as a bullet point.

Reply:
Fixed, thanks!